unattended.sh

```diff
@@ -697,6 +697,10 @@ function update_after_installation() {
 #####################################################
 ################ MAIN SCRIPT EXECUTION ##############
 
+if [[ $EUID -ne 0 ]]; then
```

Comment:
```suggestion
if [ $EUID -ne 0 ]; then
```
Use one brackets for POSIX format compatible (example pure sh)